models/aged_creditor.json

```diff
@@ -7,21 +7,21 @@
             "attributes": {
                 "code": {
                     "column": "demarcation_code",
-                    "label": "Code",
+                    "label": "Demarcation Code",
                     "type": "string",
                     "description": "Demarcation Code desc"
```

Comment:
Sure. I like. For codes I think we can say where they're defined - e.g. I'll say which codes are the treasury's. This one's the demarcation board's etc.